nautobot/docs/development/index.md

```diff
@@ -59,21 +59,38 @@ Over time, the process of moving work from _Future_ to _Near Term_ to _Current_
 
+### Versioning
+
+Nautobot adheres to the Semantic Versioning ("SemVer") strategy, which gives us versions of the format `x.y.z`.
+
+* `X` is the major release number. Contains breaking changes or switching default behavior and shadowing legacy/deprecated behavior. May contain new functionality and bug fixes as well.
+* `Y` is the minor release number. Contains new functionality and bug fixes. May introduce deprecation warnings but will not remove or change default behavior.
+* `Z` is the patch release number. Will only contain bug fixes and small feature enhancements aimed at addressing user-experience bugs.
+
+For more information, please visit [SemVer.org](https://semver.org/).
+
 ### Release Schedule
 
-Here is what you need to know about Nautobot releases:
+Nautobot aims to publish against following release schedule:
+
+* One (1) major release per year
+* Three (3) minor releases per year
+* At minimum one (1) patch release every two weeks or more frequently as needed.
 
-* The initial launch of Nautobot is version 1.0.0beta1 (major.minor.patch) released on February 24, 2021.
-* The core team estimates quarterly releases with the majority of them being minor releases.
-* It is an aspirational goal that there will be no more than one major release per year as major releases do indicate a break in backwards compatibility.
-* For information pertaining to patch releases, which will be released on a schedule, please see the [Patch Releases](#patch-releases) section below.
-* Given the core team is estimating quarterly releases, there will not be firm dates for releases.
-* In order to provide more visibility into the development and release schedule of Nautobot, there will be structured notifications as follows:
-    * At the start of a release cycle, the estimated timeframe for release will be a 4-6 week window.
-    * Halfway through the release cycle (~6 weeks), the estimated timeframe for release will be narrowed to a 3-4 week window
-    * After 8-9 weeks within the development cycle, the estimated timeframe for release will be narrowed further to a 2 week window.
-    * The final notification will be provided 3-5 days before the release drops.
-* The dates and notifications will occur by updating the GitHub Release Milestone and on Slack.
+Using `2.0` as a release cycle example, that would mean we would publish:
+
+* `2.0.z`
+* `2.1.z`
+* `2.2.z`
+* `2.3.z`
+
+After the third minor release (`2.3.0`), we would plan for another major release, in this case `3.0.0`.
+
+We estimate non-patch releases will be released quarterly but dates are not set in stone. Dates and notifications will occur by updating the GitHub Release Milestone and on Slack.
+
+An example of the release timeline for the `2.0` release cycle:
+
+![Visual diagram of Nautobot release cadence.](../media/development/index_release_cadence.jpg)
```

Comment:
This diagram is great! But the font size is really tiny. Can we please bump it up a little bit?